internal/repositories/reward/rewards.go

```diff
@@ -276,3 +283,8 @@ func (r *Repository) PaginateGetEarningsByUsersDevices(ctx context.Context, user
 	userDeviceEarnings.History.PageInfo = afd.PageInfo
 	return userDeviceEarnings.History, nil
 }
+
+// divide by 10^18 to get token value.
+func weiToToken(wei types.Decimal) *decimal.Big {
+	return new(decimal.Big).Quo(wei.Big, weiVal)
```

Comment:
My guess is the precision in `weiVal` is intended to give this max precision (the library appears to default to 16?), but in testing what appears to matter is the precision of the instance on which you're calling `Quo`.

https://go.dev/play/p/PjWoBg-HCWa

This is an extremely minor point. Feel free to ignore.

Reply:
Yeah, you are precisely right I was trying to give a little more precision (IDKY) by setting it on weiVal. Though I think we can use the default of 16. Since we only need a precision of 11 `max token count == billion tokens (9 places) + 2 decimals`

Reply:
It's unclear to me that setting it to 20 on the denominator is _doing_ anything.

https://go.dev/play/p/06Xgl71EUGD

Reply:
it's not and if it was I think the default would still suffice for our needs.